.github/workflows/release-wallet.yml

```diff
@@ -0,0 +1,247 @@
+name: Deploy interface and release extension
+on:
+  workflow_dispatch:
+    inputs:
+      REACT_APP_NAMADA_ALIAS:
```

Comment:
Since the faucet PR was just recently merged, we could include those env vars as inputs (as we'll need the faucet address & limit set when connected to testnet):
```
## From .env.sample
REACT_APP_NAMADA_FAUCET_ADDRESS=atest1v4ehgw368pprgvjygv652w2yg3znvd69xucnvde3xaz5ysenx5myx3f389z5yv3nxg65xv3kx0fkez
REACT_APP_NAMADA_FAUCET_LIMIT=1000
```

Reply:
Ok, I will add the two variables on with these defaults :+1: